dev-tools/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+
+RUN apk add python3 \
```

Comment:
`python3` is not required here as it is already installed by the app-builder image and even if it wasn't, it would be installed by `py3-qt5` anyway.
Please order all the packages in this list by alphabetical order as well.

Reply:
Indeed.